contrib/docker-entrypoint.sh

```diff
@@ -12,15 +12,14 @@ if [ $(id -u) = 0 ]; then
 fi
 
 if [ "$1" = 'test' ]; then
-  eval $CMD_PREFIX /home/pganalyze/collector --test --no-log-timestamps --no-reload
+  shift
```

Comment:
Are there any compatibility problems with this if `/bin/sh` is not bash? (should we use `/bin/bash` instead, assuming that's supported on Alpine?)

Reply:
I checked this by creating a simple shell script and make sure that it worked with `/bin/sh`. Also Keiko 2023 Nov is saying "alpine image doesn't have bash (quick tested and it didn't work)" so I think it's okay to keep this way